### ISSUE TITLE
PhantomJS is dead. Any plans for the future?

### ISSUE THREAD

**Issue by SteadyGiant** [cpsievert/rdom#17]
I saw the sole dev of phantomJS has [stepped down](https://semaphoreci.com/blog/2018/03/27/phantomjs-is-dead-use-chrome-headless-in-continuous-integration.html) and 2.1.1 is the last stable version. I was just wondering, do you have any plans for the future of `rdom`, or do you want to keep this package as-is in perpetuity? This package really helped me in the past and I was just curious.

Edit: Oops. I just found the `decapitated` package for using Headless Chrome with R.

Edit2 (2019-07-31): It looks like the new [Chromote](https://twitter.com/jcheng/status/1156625993412382720) package by RStudio is designed to replace phantomJS as a dependency for their packages. Maybe relevant for `rdom` moving forward.

**Comment by muschellij2** [comment]
I'd add that it's likely now at https://github.com/RLesur/crrri

**Comment by SteadyGiant** [comment]
Yeah I just saw today [even the creator of decapitated uses crrri](https://twitter.com/hrbrmstr/status/1156557419708899333). Thanks.
